lldb/source/DataFormatters/TypeSummary.cpp

```diff
@@ -201,3 +216,12 @@ std::string ScriptSummaryFormat::GetDescription() {
   }
   return std::string(sstr.GetString());
 }
+
+std::string ScriptSummaryFormat::GetName() { return m_script_formatter_name; }
+
+std::string ScriptSummaryFormat::GetSummaryKindName() {
+  if (!m_python_script.empty())
```

Comment:
Don't we alway need a `m_python_script`? When can this be empty?

Reply:
There was an existing check in the constructor, so I follow suite here. Doesn't lldb support Lua as well?

Reply:
Not sure what the difference between a "python" and "script" is? Lua is a supported scripting language, but it isn't used for any data formatting or summary providers.